app/controllers/staff/matched_efforts_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Staff::MatchedEffortsController < Staff::BaseController
+  def new
+    @activity = Activity.find(params[:activity_id])
+    authorize @activity
```

Comment:
These `authorize` calls are going to invoke `ActivityPolicy#new?` and `ActivityPolicy#create?` -- is that what we want? This is about attaching new records to an activity, not creating an activity itself.